medical_prescription_us/models/medical_prescription_order_line.py

```diff
@@ -43,21 +46,25 @@ class MedicalPrescriptionOrderLine(models.Model):
     @api.model
     def create(self, vals):
         if not vals.get('date_stop_treatment'):
+            date_start = fields.Datetime.from_string(
```

Comment:
@lasley The changes requested to move date_start and medicament outside of the `try` block is causing several issues it seems, as described in the previous comment. Is it okay if we defer these changes until 10.0 and move to the roadmap instead here?